apps/src/lib/node/ledger/shell/prepare_proposal.rs

```diff
@@ -36,6 +36,11 @@ mod prepare_block {
             &mut self,
             req: RequestPrepareProposal,
         ) -> response::PrepareProposal {
+            tracing::info!(
```

Comment:
We already get a log for this from tendermint.

Reply:
:+1: I've removed this